scripts/publishNPMPackages.js

```diff
@@ -32,6 +32,7 @@ import { invariant } from '@adeira/js';
       '@adeira/flow-bin',
       '@adeira/eslint-config',
       '@adeira/global-id',
+      '@adeira/bc-checker',
```

Comment:
Alphabetical order would be perhaps nicer... :)

Reply:
🙈